specs/static/bytecode/fjord-gas-price-oracle-deployment.txt

```diff
@@ -0,0 +1 @@
15ec565b61139290600a6117b0565b61139c82846115ec565b610b959190611629565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b6000602082840312156113e757600080fd5b813567ffffffffffffffff808211156113ff57600080fd5b818401915084601f83011261141357600080fd5b813581811115611425576114256113a6565b604051601f82017fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe0908116603f0116810190838211818310171561146b5761146b6113a6565b8160405282815287602084870101111561148457600080fd5b826020860160208301376000928101602001929092525095945050505050565b600060208083528351808285015260005b818110156114d1578581018301518582016040015282016114b5565b818111156114e3576000604083870101525b50601f017fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe016929092016040019392505050565b60006020828403121561152957600080fd5b5035919050565b60006020828403121561154257600080fd5b5051919050565b60006020828403121561155b57600080fd5b815173ffffffffffffffffffffffffffffffffffffffff81168114610aba57600080fd5b60006020828403121561159157600080fd5b815163ffffffff81168114610aba57600080fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b600082198211156115e7576115e76115a5565b500190565b6000817fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff0483118215151615611624576116246115a5565b500290565b60008261165f577f4e487b7100000000000000000000000000000000000000000000000000000000600052601260045260246000fd5b500490565b600063ffffffff80831681851681830481118215151615611687576116876115a5565b02949350505050565b600181815b808511156116e957817fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff048211156116cf576116cf6115a5565b808516156116dc57918102915b93841c9390800290611695565b509250929050565b600082611700575060016105c9565b8161170d575060006105c9565b8160018114611723576002811461172d57611749565b60019150506105c9565b60ff84111561173e5761173e6115a5565b50506001821b6105c9565b5060208310610133831016604e8410600b841016171561176c575081810a6105c9565b6117768383611690565b807fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff048211156117a8576117a86115a5565b029392505050565b6000610aba83836116f1565b6000808212827f7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff038413811516156117f6576117f66115a5565b827f800000000000000000000000000000000000000000000000000000000000000003841281161561182a5761182a6115a5565b50500190565b60007f7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff600084136000841385830485118282161615611871576118716115a5565b7f800000000000000000000000000000000000000000000000000000000000000060008712868205881281841616156118ac576118ac6115a5565b600087129250878205871284841616156118c8576118c86115a5565b878505871281841616156118de576118de6115a5565b505050929093029392505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b60007fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff820361194c5761194c6115a5565b506001019056fea164736f6c634300080f000a
```

Comment:
Where is the source code for this? I don't see it [here](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L2/GasPriceOracle.sol)

Reply:
It's the bytecode of the updated GasPriceOracle from this [PR](https://github.com/ethereum-optimism/optimism/pull/9618/files#diff-7eeb2cdd3a766512a162b2921f45ae64a3ae91f99d7d408ca11defb6142f4ba5).

You can verify it by grabbing the latest commit/running forge build and grabbing the bytecode via:

```
cat forge-artifacts/GasPriceOracle.sol/GasPriceOracle.json | jq ".bytecode.object"
```